libindy/src/services/pool/types.rs

```diff
@@ -465,6 +465,27 @@ impl PoolConfig {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct PoolOpenConfig {
```

Comment:
It is better to have in domain as changes can count backward compatibility problems